playwright/all_tests/instruction/instructionFilters.spec.ts

```diff
@@ -1,52 +1,54 @@
-import { expect, Page, test } from '@playwright/test'
+import { expect, Locator, Page, test } from '@playwright/test'
 import { loginTestGroup, Role, setMultiSelectDropdownOptions, setSingleSelectDropdownOption } from '../../helpers'
 import { Exam } from 'web/src/types'
 
-loginTestGroup(test, Role.YLLAPITAJA)
-
 const testNameMapping = {
   [Exam.SUKO]: (number: number) => `Test name ${number} FI`,
   [Exam.LD]: (number: number) => `LD Test name ${number} FI`,
   [Exam.PUHVI]: (number: number) => `PUHVI Test short description ${number} FI`
 }
 
-const getExpectedNames = (numbers: number[], exam: Exam): string[] => numbers.map(testNameMapping[exam])
+const mapPromiseAll =
+  <T, U>(fn: (arg: T) => Promise<U>) =>
+  (args: T[]) =>
+    Promise.all(args.map(fn))
 
-const getTextBasedOnExam = (listItem: any, exam: Exam): Promise<string> => {
-  const testId = exam === Exam.SUKO ? 'instruction-title' : 'instruction-body'
-  return listItem.getByTestId(testId).innerText()
-}
+const getTestId = (exam: Exam) => (exam === Exam.SUKO ? 'instruction-title' : 'instruction-body')
```

Comment:
Miks SUKOlle käytetään eri testId:tä?

Reply:
Homma taitais yksinkertaistua, jos `testNameMapping` sijaan ois `bodyTextByExam`, ja käyttäis aina testId:nä `instruction-body`?